src/App/Routes/ContributionDetailsPage/index.tsx

```diff
@@ -56,34 +56,42 @@ const ContributionDetailsPageContainer: FC = () => {
       setDisplayRegisterModal(true);
       return;
     }
-    setApplying(true);
+    const backEndApply = config.FEATURE_BACKEND_APPLY == "true";
 
-    toastPromise(applicationRepository.create({ contributionId: contribution.id, contributorId }), {
-      success: () => {
-        return (
-          <div className="leading-[1rem] line-clamp-3">
-            Thank you for your application for{" "}
-            <Link to={`/contributions/${contribution.id}`} className="italic underline">
-              {contribution.title}
-            </Link>
-            , we'll review it and get in touch with you very shortly!
-          </div>
-        );
-      },
-      pending: () => "Your application is being processed",
-      error: () => (
-        <>
-          An error occured while appliying to this contribution
-          <br />
-          Please try again
-        </>
-      ),
-    });
+    if (backEndApply) {
```

Comment:
The toggle should also change the check done here https://github.com/onlydustxyz/marketplace-frontend/pull/109/files#diff-5063f0e6abfcf27a665ab0a9fafc1e6d3010e23fc97b7a398cd73f01bbf8791cR54
When true, check for userDiscordHandle to be undefined, when false, don't check at all since it will be requested in the typeform